assets/js/order-script.js

```diff
@@ -27,49 +30,49 @@ jQuery(document).ready(function($) {
 
 			window.open(url,'_blank');
 		}
-	});
+	} );
 
-	$('#wpo_wcpdf-data-input-box').insertAfter('#woocommerce-order-data');
+	$( '#wpo_wcpdf-data-input-box' ).insertAfter('#woocommerce-order-data');
 	
 	// enable invoice number edit if user initiated
-	$( ".wpo-wcpdf-set-date-number, .wpo-wcpdf-edit-date-number, .wpo-wcpdf-edit-document-notes" ).on("click", function() {
-		$form = $(this).closest('.wcpdf-data-fields');
+	$( document ).on( 'click', '.wpo-wcpdf-set-date-number, .wpo-wcpdf-edit-date-number, .wpo-wcpdf-edit-document-notes', function() {
+		let form = $(this).closest('.wcpdf-data-fields');
```

Comment:
form is a jQuery object, we prefix all jQuery objects with $ in our plugin scripts